### PR TITLE
COMMERCE-4766 ForceSync for Product Specification Options Portlet, to…

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
… ensure refreshed list

*Reminder*
The fix is based on a deprecated method, we might want to refactor this later.

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-options-web/src/main/java/com/liferay/commerce/product/options/web/internal/portlet/CPSpecificationOptionsPortlet.java [hunk 1]
```diff
@@ -63,6 +67,20 @@
...
 
+	@Override
+	public void processAction(
+			ActionRequest actionRequest, ActionResponse actionResponse)
+		throws IOException, PortletException {
+
+		try (ProxyModeThreadLocalCloseable proxyModeThreadLocalCloseable =
+				new ProxyModeThreadLocalCloseable()) {
+
+			ProxyModeThreadLocal.setForceSync(true);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-options-web/src/main/java/com/liferay/commerce/product/options/web/internal/portlet/CPSpecificationOptionsPortlet.java:78]: we might be able to replace it with this
```
try (SafeClosable safeClosable =
		ProxyModeThreadLocal.setWithSafeClosable(true)) {
```

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by dsitu** [comment]
Closing sent directly here https://github.com/brianchandotcom/liferay-portal/pull/94158.

**Pull request closed without merging**